noir-projects/noir-contracts/contracts/contract_class_registerer_contract/src/main.nr

```diff
@@ -33,13 +33,28 @@ contract ContractClassRegisterer {
 
     #[private]
     fn register(artifact_hash: Field, private_functions_root: Field, public_bytecode_commitment: Field) {
-        // TODO: Validate public_bytecode_commitment is the correct commitment of packed_public_bytecode
         // TODO: Validate packed_public_bytecode is legit public bytecode
+        // TODO: We should be able to remove public_bytecode_commitment from the input if it's calculated in this function
 
         // docs:start:pop_capsule
         let packed_public_bytecode: [Field; MAX_PACKED_PUBLIC_BYTECODE_SIZE_IN_FIELDS] = pop_capsule();
         // docs:end:pop_capsule
 
+        // First field element contains the length of the bytecode
+        let bytecode_length_in_bytes: u32 = packed_public_bytecode[0] as u32;
```

Comment:
I guess we could eventually change this and pass the length as an input to register?